pom.xml

```diff
@@ -22,7 +22,7 @@ under the License.
 
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.apache.reef</groupId>
-    <version>0.10-incubating-SNAPSHOT</version>
```

Comment:
In prior (non-Apache) releases, I used the maven release plugin for this. Is that an option now as well?